sway/input/input-manager.c

```diff
@@ -82,6 +83,75 @@ static bool input_has_seat_configuration(struct sway_input_manager *input) {
 	return false;
 }
 
+static void sway_input_manager_libinput_config_pointer(struct sway_input_device *input_device) {
```

Comment:
Do we know this is only pointer configuration or does this apply to any other devices like tablet tool or touch devices?

Reply:
setting like accel don't make sense to me for non-pointers. I have no idea about tap and other settings though, but I check that the input device is of type WLR_*_POINTER before calling this function... 

According to https://wayland.freedesktop.org/libinput/doc/latest/group__config.html#details we seem to only set options that make sense for pointers and touchpads (which is pointer for us), but there are functions to probe if the device is capable so we could remove the wlr type check and use these instead.
I think it's a bit more ambitious than just reviving the sway-input feature though, probably should get its own ticket/PR if we want to be that modular?

Reply:
Touch devices and tablet tool devices will now be treated separately. Look at the rootston code for a preview of what touch and tablet tool support will look like.

If this configuration applies to tablet tool and touch, then it should be applied to those kinds of devices as well.